pub/index.php

```diff
@@ -10,7 +10,7 @@
 use Magento\Framework\App\Filesystem\DirectoryList;
 
 try {
-    require realpath(__DIR__) . '/../app/bootstrap.php';
```

Comment:
On closer look I'd say that your suggestion is not necessary. Require deals fine with such paths, it does not need any further assistance, or do you have a scenario at hand where that is not the case. I don't know of any nor can I imagine one.